backend/model/movies-model.js

```diff
@@ -0,0 +1,13 @@
+// Load mongoose package
+import mongoose from 'mongoose';
+
+// Create a schema
+const MovieSchema = new mongoose.Schema({
```

Comment:
I'm saying if we need the Movie model without having it associated to mongoose we need to separate this in two files, not sure if we ever gonna need like that that